### ISSUE TITLE
two pages -  one is scrapable and one is not

### ISSUE THREAD

**Issue by AcidkoHorkaCokolada** [kevinzg/facebook-scraper#802]
Hi everyone, 

i have two different pages. one is scrapable and one is not. 
fb.com/239366402785275 won't get any results
fb.com/313187412842466 this is ok
 

code. 
`posts = get_posts(id,  pages=10, timeout=600, options={"comments": True,"allow_extra_requests": True })`

am i doing anything wrong? whats difference?

**Comment by AcidkoHorkaCokolada** [comment]
set_cookies("cookies.json") would not get loaded..

**Comment by neon-ninja** [comment]
Yes, seems 239366402785275 requires you to be logged in to see posts. So you'll need to get your cookies working. Try open https://m.facebook.com/ministerstvovnutraSR in an incognito window, you can see there's no posts there.
